src/main/java/org/takes/rs/RsWithoutHeader.java

```diff
@@ -52,18 +53,16 @@ public RsWithoutHeader(final Response res, final CharSequence name) {
         super(
             new Response() {
                 @Override
+                @SuppressWarnings("unchecked")
```

Comment:
@krzyk, this is indeed needed as ```Collections.EMPTY_LIST``` is a generic List and ```Concat``` constructor expect an iterable with type ```String``` here.